CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ if(CMAKE_CXX_STANDARD EQUAL "98" )
 endif()
 
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 11) # Supported values are ``11``, ``14``, and ``17``.
+  set(CMAKE_CXX_STANDARD 14) # Supported values are ``11``, ``14``, and ``17``.
```

Comment:
I guess we should remove 11 from this list 😄

Reply:
Good point 😃 

Is it reasonable to match the same supported values in ITK? i.e.:
> Supported values are 14, 17, 20, and 23.

Or are there potential project reasons to not include 20 and 23 yet?

Reply:
23 is not finalized yet, so it is better not to include it. I think 20 was not finalized when this list was originally created, and there is no reason for its absence now.